### PR TITLE
Add 2 tests for CitizenUpdateCaseStateAAT event

### PR DESCRIPTION
**Pull request by danielbe11**

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/divorce/citizen/event/CitizenUpdateCaseStateAatTest.java [hunk 1]
```diff
@@ -73,4 +75,41 @@ void shouldNotUpdateCaseStateWhenStateIsInvalid() {
...
+
+        caseData.setApplicant2(new Applicant());
+        caseData.getApplicant2().setMiddleName("Holding");
+
+        caseDetails.setData(caseData);
+        caseDetails.setId(caseId);
+
+        final AboutToStartOrSubmitResponse<CaseData, State> response = citizenUpdateCaseStateAat.aboutToSubmit(caseDetails, caseDetails);
+
+        assertThat(response.getData().getApplicant2().getMiddleName()).isEqualTo(EMPTY_STRING);
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/citizen/event/CitizenUpdateCaseStateAatTest.java:93]: you can use assertThat(response.getData().getApplicant2().getMiddleName()).isEmpty() ?

Comment (danielbe11) [reply src/test/java/uk/gov/hmcts/divorce/citizen/event/CitizenUpdateCaseStateAatTest.java]: done thanks

File: src/test/java/uk/gov/hmcts/divorce/testutil/TestConstants.java [hunk 2]
```diff
@@ -65,6 +65,8 @@
     public static final String SIGN_IN_DISSOLUTION_TEST_URL = "dissolutionTestUrl";
     public static final String APPLICANT_2_SIGN_IN_DISSOLUTION_TEST_URL = "applicant2DissolutionTestUrl";
 
+    public static final String EMPTY_STRING = "";
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/divorce/testutil/TestConstants.java:68]: Can remove this now ?

**Review by nitinprabhu — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/divorce/testutil/TestConstants.java (hunk 2 above)
Comment (danielbe11) [reply src/test/java/uk/gov/hmcts/divorce/testutil/TestConstants.java:68]: Use it here https://github.com/hmcts/nfdiv-case-api/pull/1275/files#diff-f8a9d4709a207b60304bc4fa98c1048e55d6df00e49159036c6dfde3b9213d62R103

**Pull request merged**